migration/sql-files/001-audit-logs.sql

```diff
@@ -0,0 +1,25 @@
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+
+-- event types
+CREATE TABLE event_types (
+    id uuid primary key DEFAULT uuid_generate_v4() NOT NULL,
+    name varchar
+);
+
+insert into event_types (id, name) values ('7aea0277-d6fa-4df9-8224-a27fa4096ec7', 'user_search');
+
+-- index to query event type by name, which must be unique
+CREATE UNIQUE INDEX uix_event_type_name ON event_types USING btree (name);
+
+-- audit logs
+CREATE TABLE audit_logs (
+    id uuid primary key DEFAULT uuid_generate_v4() NOT NULL,
+    created_at timestamp with time zone NOT NULL default now(),
```

Comment:
We could keep `deleted_at` since it gives out-of-the-box soft delete support by the ORM layer.

Reply:
no, we agreed on the schema design that audit records cannot be deleted, so there's no reason to support soft-deletes